vars/runPipeline.groovy

```diff
@@ -222,7 +222,6 @@ def testDownstreamProject (name) {
 
                 case 'sociomantic-tsunami/ocean':
                     sh '''
-                    echo '\nTEST_FILTER_OUT += $C/test/sysstats/main.d' >> Build.mak
```

Comment:
FYI for the other people looking at this:
Disabled in https://github.com/dlang/ci/pull/91, fixed in https://github.com/sociomantic-tsunami/ocean/pull/386